docs/documentation/development/bindings/categories.md

```diff
@@ -14,7 +14,44 @@ The thing type definition allows to specify a category. User interfaces can pars
 
 | Category        | Description                                          |
 |-----------------|------------------------------------------------------|
-
+| Aircondition    | Air condition devices incl. Fans |
+| BinarySensor    | General purpose sensor that returns a boolean value, i.e. a contact that reports open/close on a door or window |
+| Bridge          | Bridges/Gateway need to access other devices like used by Philips Hue for example |
+| Boiler          | Devices that heat up water |
+| Camera          | All kinds of cameras |
+| Car             | Smart Cars |
+| Cistern         | Cistern |
+| Clock           | Devices that provide the current time |
+| Coffee          | Coffee machines |
+| Fridge          | Fridges, Freezers |
+| Furniture       | Smart furniture |
+| GarageDoor      | Garage doors |
+| GeneralSensor   | General purpose sensor that does not fit in any category |
+| Heating         | Devices that deal with heating |
+| Hifi            | Devices related to TV/audio, such as satelite or cable receivers, recorders, etc.
+| HomeApplicance  | Devices that look like Waschingmachines, Dishwashers, Dryers, etc. |
+| Light           | Devices that illuminate something, such as bulbs, etc. |
+| Lock            | Devices whose primary pupose is locking something |
+| Microphone      | Devices to record sound |
+| Mixer           | Kitchen devices for making cake |
+| MotionSensor    | Motion sensors/detectors |
+| MultiSwitch     | Device which controls MULTIPLE states (else see Switch) of something |
+| Oven            | Ovens, microwaves, etc. |
+| Phone           | Anything related to telephony, such as mobile phones, landline phones, etc. |
+| PowerSocketDevice | Small devices to be plugged into a power socket in a wall which stick there |
+| Projector       | Devices that project a picture somewhere |
+| Pump            | Water pumps |
+| Radiator        | Radiators that heat up rooms |
+| Robots          | Vacuum cleaners such as cleaning robots, lawn mowers, etc. |
```

Comment:
Might also be a walking humanoid. Difficult to have such a general category. How do others do it?